product-features/liquidity-providers/faqs.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Common questions you might have as a liquidity provider.
+---
+
+# FAQs
+
+<details>
+
+<summary>What is the difference between swap fee and fee rebate?</summary>
+
+The **swap fee** is the total fee charged to users for executing a token swap. It's the fee that directly impacts the trader and is displayed as "Fees" in the Pool Info panel.
+
+On the other hand, the **fee rebate** is the portion of the swap fee that is distributed to liquidity providers as a reward for supplying liquidity to the pool. The remaining portion of the swap fee goes to the ALEX platform. You can also find the fee rebate percentage in the Pool Info panel.
```

Comment:
ALEX platform change to ALEX Lab Foundation

Reply:
Done